Source/WebCore/css/CSSComputedStyleDeclaration.cpp

```diff
@@ -3624,6 +3624,10 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
             return valueForContainIntrinsicSize(style, style.containIntrinsicWidthType(), style.containIntrinsicWidth());
         case CSSPropertyContainIntrinsicHeight:
             return valueForContainIntrinsicSize(style, style.containIntrinsicHeightType(), style.containIntrinsicHeight());
+        case CSSPropertyContentVisibility:
+            if (!m_element->document().settings().cssContentVisibilityEnabled())
+                return nullptr;
+            return CSSPrimitiveValue::create(style.contentVisibility());
```

Comment:
Is there a way to do this and take advantage of `cssValuePool`? Should help performance if we can.

Reply:
You are right, I see how that is more efficient. I went with your original idea, I guess we can implement that idea more in the future.